shell/detail/workload/index.vue

```diff
@@ -133,13 +137,15 @@ export default {
     },
 
     podTemplateSpec() {
-      const isCronJob = this.value.type === WORKLOAD_TYPES.CRON_JOB;
-
-      if ( isCronJob ) {
+      if ( this.value.type === WORKLOAD_TYPES.CRON_JOB ) {
         return this.value.spec.jobTemplate.spec.template.spec;
-      } else {
-        return this.value.spec?.template?.spec;
       }
+
+      if ( this.value.type === WORKLOAD_TYPES.POD ) {
```

Comment:
This check should be removed because the contents of the conditional statement applies to all workload types except CronJobs (Deployments, StatefulSets etc. all have pod specs).

Reply:
This is for when workload creation type is pod.

More info: `pod` https://github.com/rancher/dashboard/pull/6632#discussion_r959773692

Pod spec: https://kubernetes.io/docs/concepts/workloads/pods/